woocommerce/wc-admin
# Modify table download button styling

Fixes #

Replace the current download button with the new design from the tabular data component.

### Screenshots

<img width="258" alt="screen shot 2018-10-17 at 5 52 18 pm" src="https://user-images.githubusercontent.com/10561050/47118521-7dc59300-d235-11e8-9db6-aac3ec9c64b1.png">

### Detailed test instructions:

1.  Visit `/wp-admin/admin.php?page=wc-admin#/analytics/revenue`
2.  Confirm that the download button is styled similar to screenshot above.
